packages/core/src/lib/filter/subscription.ts

```diff
@@ -0,0 +1,286 @@
+import { Stream } from "@libp2p/interface/connection";
+import {
+  CustomEvent,
+  EventEmitter,
```

Comment:
I think these interfaces are default to browser. We can avoid using these it seems

Reply:
I see you resolved it - do you think it is better to keep using copy of it from `libp2p`?